lib/polleverywhere/models.rb

```diff
@@ -154,6 +160,16 @@ def self.get(permalink)
         from_hash(:permalink => permalink).fetch
       end
 
+      def make_current
+        http.post(:permalink => permalink).to("/my/polls/current").response do |response|
```

Comment:
Activation isn't part of our publicly supported API, so I don't think we'll be able to merge this part of the pull request at this time.

Reply:
Thought I'd try :-)  Do you want a new pull req with just the fixes to Poll#all?